docs/asset-management.rst

```diff
@@ -76,43 +166,43 @@ Make sure you are accessing the CodeChain port. In this example, it is assumed t
 
     const sdk = new SDK({ server: “http://localhost:8080” });
 
-In this example, it is assumed that there is something that created a parcel out of the transactions. sendTransaction has been declared for later use.
+The MemoryKeyStore is created for testing purposes. In real applications, the MemoryKeyStore would probably be some sort of storage, or something similar,
+that would hold and manage the key pair(private and public keys). The P2PKH is responsible for locking and unlocking scripts.
 ::
 
-    // sendTransaction() is a function to make transaction to be processed.
-    async function sendTransaction(tx) {
-        const parcelSignerSecret = "ede1d4ccb4ec9a8bbbae9a13db3f4a7b56ea04189be86ac3a6a439d9a0a1addd";
-        const parcelSignerAddress = SDK.util.getAccountIdFromPrivate(parcelSignerSecret);
-        const parcel = sdk.core.createChangeShardStateParcel({
-            transactions: [tx],
-            nonce: await sdk.rpc.chain.getNonce(parcelSignerAddress),
-            fee: 10,
-        }).sign(parcelSignerSecret)
-        return await sdk.rpc.chain.sendSignedParcel(parcel);
-    }
-
-Each users need an address for them to receive/send assets to. Addresses are created by the assetAgent.
+    const keyStore = await sdk.key.createMemoryKeyStore();
+    const p2pkh = await sdk.key.createP2PKH({ keyStore });
+
+Each user needs an address for them to receive/send assets to. Addresses are created by p2pkh. In this example, Bob's address is created differently,
```

Comment:
What do you mean by "Bob's address is created differently"? It is the address we got from Bob.